### Post 77
**Post URL**: /t/project-1-llm-based-automation-agent-discussion-thread-tds-jan-2025/164277/77
- **ID**: 592061
- **Author**: Telvin Varghese (22f2001640)
- **Created At**: 2025-02-08T08:33:27.497Z
- **Reply To**: Post 76 (Guddu Kumar Mishra , 22f3001315)
- **Content**:  
  <a class="mention" href="/u/s.anand">@s.anand</a> <a class="mention" href="/u/carlton">@carlton</a> <a class="mention" href="/u/jivraj">@Jivraj</a><br>
<strong>I am getting below error message from LLM end points <strong><a href="https://api.openai.com/v1/chat/completions" rel="noopener nofollow ugc">https://api.openai.com/v1/chat/completions</a> or <a href="https://aiproxy.sanand.workers.dev/openai/v1/embeddings" rel="noopener nofollow ugc">https://aiproxy.sanand.workers.dev/openai/v1/embeddings</a></strong> , while running my project .</strong>
<img src="https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/7/7/775bdd56ec848f8c87546375952710aacc722ba1.png" alt="image" data-base62-sha1="h1TDysFxS8sSBnC3Yk9Z4t6w7u1" width="690" height="25" data-dominant-color="303030"><br>
Kindly help me to resolve this issue. <img src="https://emoji.discourse-cdn.com/google/cry.png?v=12" title=":cry:" class="emoji" alt=":cry:" loading="lazy" width="20" height="20">
- **Reactions**: None
- **Post Number**: 77

